content/security/improvements.adoc

```diff
@@ -0,0 +1,66 @@
+---
+layout: security
+title: "Improvements by the Security Team"
+section: security
+---
+
+These are some contributions by members of the Jenkins security team that weren't delivered as security fixes, but still are security-related.
+
+
+== 2020
+
+Jenkins (core)::
+- Plugin Manager: https://github.com/jenkinsci/jenkins/pull/4553[Add inline security warnings to installed plugins list]
```

Comment:
It's not intended as an overview of security things happening in Jenkins for a user/admin audience, so I don't think that would be useful.

It's intended as a reference for what the contributions from the security team are outside what is referenced in advisories, as well as inspiration for contributors interested in security but perhaps not ready for the security team (and the latter only to an extent, otherwise we'd list changes contributed by anyone). In neither case, specific releases matter IMO.